.github/config.yml

```diff
@@ -0,0 +1,27 @@
+# https://github.com/behaviorbot/welcome
+# combines new-issue-welcome, new-pr-welcome, and first-pr-merge
+
+# Comment on first-time issues
+# https://github.com/behaviorbot/new-issue-welcome
+newIssueWelcomeComment: >
+  👋 Thanks for opening your first issue here! If you're reporting a bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
+
+  To make it easier for us to investigate your issue, please follow the [contributing guidelines](https://github.com/nodejs/node/blob/master/doc/guides/contributing/issues.md).
+
+# Comment on PRs from first-time contributors
+# https://github.com/behaviorbot/new-pr-welcome
+newPRWelcomeComment: >
+  💚 Thanks for opening this pull request! 💚
+
+  Please make sure to update documentation and tests when appropriate, and follow the [commit guidelines](https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#commit-message-guidelines)
+
+  For non-trivial changes, pull requests must be left open for *at least* 48 hours during the week, and 72 hours on a weekend.
+
+  If this pull request changes includes code changes, the CI must be started by a collaborator.
+
+  We get a lot of pull requests on this repo, so please be patient and we will review this as soon as we can.
+
+# Comment when a first-time contributor's PR is merged
+# https://github.com/behaviorbot/first-pr-merge
+firstPRMergeComment: >
```

Comment:
I think this can be removed since we rarely use the merge button and have our own workflow. It can be added when we adopt something else (e.g. a Jenkins job that merge the PR for us)

Reply:
@zeke I am fine with leaving those texts. Although just for the sake of explanation, I am not actually aware of anybody using the button...at least not for first-time PRs. If you look at the PRs that are "merged", basically they are all opened and merged by collaborators, because right now you will need to push the HEAD that you are about to push to master to the same PR branch in order to make it purple. Unless you are the person that's going to push to master real soon, you are unlikely to get to the final state of that commit and push it to the PR branch right before it's pushed to master.

https://github.com/nodejs/node/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Amerged